### DISCUSSION TITLE
Does VSQlik cache data extracted from Sense servers?

### DISCUSSION THREAD

**Discussion by mountaindude** [q-masters/vsqlik#803] [General]
My setup is that I am connecting to a QSEoW server with a hundred or so apps. 
Nothing big in terms of app size. 

I am struggling a bit to understand when and how VSQlik refreshes data from the Sense server. 
Is it done on a global basis for all folders in the workspace? Not ideal as I might have 10-15 other folders in the workspace, some of which may be mounted over slow VPN connections. Refreshing those is slow and not something I want to trigger if I can avoid it.

Best would be if a refresh could be done for either the entire Sense server (this could take long time though for a  large server) or for individual apps. Maybe just for scripts within an app too.

I guess I am missing some docs for how this works, as a starter 😀.

**Comment by konne** [comment]
Hi,
yes the refreshing of data is at the moment not optimal. We are still on the topic to make a login via browser and stealing of the session cookie possible to get rid of all different ways to authenticate.

I will try to add in the next weeks some more docu around that topic.
